test/builder/lib/QuarkBuilderTest.sol

```diff
@@ -149,6 +172,20 @@ contract QuarkBuilderTest {
         revert("no mock weth for that chain id bye");
     }
 
+    function paycall_(uint256 chainId) internal pure returns (address) {
```

Comment:
We should call this `paycallUsdc_` or add `assetSymbol` as an argument.

Reply:
updated to `paycallUsdc_`